node/test/hyperbahn/unadvertise.js

```diff
@@ -35,7 +35,6 @@ if (require.main === module) {
 function runTests(HyperbahnCluster) {
     HyperbahnCluster.test('advertise and forward', {
         size: 5,
-        servicePurgePeriod: 50
```

Comment:
Please keep this test and create a new one in this file for unadvertise endpoint.